api/openapi/v2/system-agent.yaml

```diff
@@ -0,0 +1,634 @@
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/RequestEnvelope'
+      responses:
+        '207':
+          description: "Indicates a multi-part response supportive of accepting multiple requests at once. The 'statusCode' property of each response in the returned array will indicate success or failure."
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/correlatedResponseHeader'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  anyOf:
+                    - $ref: '#/components/schemas/ErrorResponse'
+                    - $ref: '#/components/schemas/ResponseEnvelope'
+  /config:
+    post:
+      summary: "Update the configuration for the targeted service(s)."
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              anyOf:
+                - $ref: '#/components/schemas/SetConfigRequest'
+                - type: array
+                  items:
+                    $ref: '#/components/schemas/SetConfigRequest'
+      responses:
+        '200':
+          description: "OK"
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/correlatedResponseHeader'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SetConfigResponse'
+        '207':
+          description: "Indicates a multi-part response supportive of accepting multiple requests at once. The 'statusCode' property of each response in the returned array will indicate success or failure."
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/correlatedResponseHeader'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  anyOf:
+                    - $ref: '#/components/schemas/ErrorResponse'
+                    - $ref: '#/components/schemas/SetConfigResponse'
+        '400':
+          description: "Bad request"
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/correlatedResponseHeader'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+  /config/{services}:
+    get:
+      summary: "Retrieves the current configuration of the targeted service(s)."
+      parameters:
+        - $ref: '#/components/parameters/correlatedRequestHeader'
+        - name: services
+          in: path
+          required: true
+          schema:
+            type: string
+          description: "A comma separated list of the services for which to obtain configuration settings."
```

Comment:
Simplify to support retrieval of single service's configuration.  Can use multiple use-case requests to retrieve configuration across services within a single transport request.